sdk/src/main/java/com/hcaptcha/sdk/tasks/Task.java

```diff
@@ -155,6 +168,18 @@ public Task<TResult> addOnFailureListener(@NonNull final OnFailureListener onFai
         return this;
     }
 
+    /**
+     * Remove a failure listener
+     * @param onFailureListener to be removed
+     * @return current object
+     */
+    public Task<TResult> removeOnFailureListener(@NonNull final OnFailureListener onFailureListener) {
+        if (!onFailureListeners.remove(onFailureListener)) {
+            HCaptchaLog.d("removeOnFailureListener: %1 not found and cannot be removed", onFailureListener);
+        }
```

Comment:
Do you think it is useful to log a debug message in case the listener is missing? @CAMOBAP

Reply:
I think yes, also user will not see it without `HCaptchaConfig.diagnosticLog` enabled